src/Sylius/Bundle/SettingsBundle/Resources/config/services.xml

```diff
@@ -21,33 +21,44 @@
         <parameter key="sylius.settings.form_factory.class">Sylius\Bundle\SettingsBundle\Form\Factory\SettingsFormFactory</parameter>
         <parameter key="sylius.settings.manager.class">Sylius\Bundle\SettingsBundle\Manager\SettingsManager</parameter>
         <parameter key="sylius.settings.schema_registry.class">Sylius\Bundle\SettingsBundle\Schema\SchemaRegistry</parameter>
+        <parameter key="sylius.settings.resolver.default.class">Sylius\Bundle\SettingsBundle\Resolver\DefaultResolver</parameter>
+        <parameter key="sylius.settings.resolver_registry.class">Sylius\Component\Registry\ServiceRegistry</parameter>
     </parameters>
 
     <services>
         <service id="sylius.controller.settings" class="%sylius.controller.settings.class%">
             <call method="setContainer">
-                <argument type="service" id="service_container" />
+                <argument type="service" id="service_container"/>
             </call>
         </service>
 
         <service id="sylius.settings.form_factory" class="%sylius.settings.form_factory.class%">
-            <argument type="service" id="sylius.settings.schema_registry" />
-            <argument type="service" id="form.factory" />
```

Comment:
In this file both are both used, I think it is good to stick with one or the other. In the rest of Sylius a space after an empty tag is used. Or do you prefer fixing those things in a seperate PR? If you still want to I can revert it.